src/publish.proj

```diff
@@ -23,5 +26,33 @@
   <Target Name="Build" DependsOnTargets="CreateContainerName">
     <!-- skip uploadToAzure when there are no nupkgs -->
     <CallTarget Targets="UploadToAzure" Condition="'@(_PackagesToPublish)' != ''" />
-  </Target>  
-</Project>
+  </Target>
+  
+  <PropertyGroup>
+    <PackageDownloadDirectory Condition="'$(DownloadDirectory)' == ''">$(PackagesDir)AzureTransfer\$(ConfigurationGroup)</PackageDownloadDirectory>
+    <FinalPublishPattern>$(PackageDownloadDirectory)\**\*.nupkg</FinalPublishPattern>
+    <FinalSymbolsPackagesPattern>$(PackageDownloadDirectory)\**\*.symbols.nupkg</FinalSymbolsPackagesPattern>
+  </PropertyGroup>
+  
+  <Target Name="PublishToAzureBlobFeed" >
+    <ItemGroup>
+      <ItemsToPush Include="$(FinalPublishPattern)" Exclude="$(FinalSymbolsPackagesPattern)" />
+    </ItemGroup>
+    <Error Condition="'@(ItemsToPush)'==''" Text="ItemsToPush for packages is empty." />
+    <PushToBlobFeed ExpectedFeedUrl="$(ExpectedFeedUrl)"
+                    AccountKey="$(AccountKey)"
+                    ItemsToPush="@(ItemsToPush)"
+                    Overwrite="$(PublishOverwrite)" />
+  </Target>
+  
+  <Target Name="PublishSymbolsToAzureBlobFeed" Condition="'$(PublishSymbols)' == 'true'" >
+    <ItemGroup>
+      <ItemsToPush Include="$(FinalSymbolsPackagesPattern)" />
+    </ItemGroup>
+    <Error Condition="'@(ItemsToPush)'==''" Text="ItemsToPush for symbols is empty." />
```

Comment:
Consider checking @(ItemsToPush->Count()) instead

Reply:
I always prefer the way he currently has the check, but it is just a preference I'm not sure if one is more efficient then the other.